bpy-build.yaml

```diff
@@ -2,22 +2,18 @@ addon_folder: MCprep_addon
 build_name: MCprep_addon
 
 install_versions:
-  - '4.0'
-  - '3.6'
-  - '3.5'
-  - '3.4'
-  - '3.3'
-  - '3.2'
-  - '3.1'
-  - '3.0'
-  - '2.93'
-  - '2.90'
-  - '2.80'
+  - 4.0
+  - 3.6 
+  - 3.5
+  - 3.4
+  - 3.3
+  - 3.2
+  - 3.1
+  - 3.0
+  - 2.93
+  - 2.9
+  - 2.83
+  - 2.8
```

Comment:
Does this still work for 2.80 with this naming convention? If so, no complaints

Reply:
Per [this comment](https://github.com/Moo-Ack-Productions/bpy-build/pull/6#pullrequestreview-1859500659), seems to not work for 2.8 (or 2.80) with the current branch of v0.3.0 of bpy-build